lib/node/rpc.js

```diff
@@ -719,6 +724,63 @@ class RPC extends RPCBase {
     return await this.headerToJSON(entry);
   }
 
+  async getCFilter(args, help) {
+    if (help || args.length < 1 || args.length > 2)
+      throw new RPCError(errs.MISC_ERROR, 'getcfilter "hash" type');
+
+    if (!this.node.bip157)
+      throw new RPCError(errs.MISC_ERROR, 'Compact filter indexing disabled');
+
+    const valid = new Validator(args);
+    const hash = valid.rhash(0);
+    const type = valid.u32(1);
+
+    if (!hash)
+      throw new RPCError(errs.MISC_ERROR, 'Invalid block hash.');
+
+    if (typeof type !== 'number')
+      throw new RPCError(errs.MISC_ERROR, 'Invalid filter type.');
+
+    if (Object.values(constants.FILTERS).indexOf(type) === -1)
+      throw new RPCError(errs.MISC_ERROR, 'Unknown filter type.');
```

Comment:
Since there is only one type right now, I wonder if this can default to `0x00` and not require the additional argument?

Reply:
I guess we can do that.